src/config.c

```diff
@@ -3177,6 +3177,19 @@ static int applyClientMaxMemoryUsage(const char **err) {
     return 1;
 }
 
+#define DB_HASH_SEED_MAX_LEN 16
+static int isValidDbHashSeed(char *val, const char **err) {
```

Comment:
```suggestion
static int isValidDbHashSeed(sds val, const char **err) {
```
I think this is the right signature. You can use sdslen below as well.

Reply:
yeah agreed, I have updated!